lib/src/data/model/movie_model.dart

```diff
@@ -0,0 +1,60 @@
+class MovieModel {
+  final String movieTitle;
+  final String originalTitle;
+  final String language;
+  final String dateRelease;
+  final String movieOverview;
+  final double voteAverage;
+  final List<int> genres;
+  final String imagePath;
+  final String backdropPath;
+  final int voteCount;
+
+  MovieModel({
+    required this.movieTitle,
+    required this.originalTitle,
+    required this.language,
+    required this.dateRelease,
+    required this.movieOverview,
+    required this.voteAverage,
+    required this.genres,
+    required this.imagePath,
+    required this.backdropPath,
+    required this.voteCount,
+  });
+
+  factory MovieModel.fromJson(
+    Map<String, dynamic> movieData,
+  ) {
+    return MovieModel(
+      movieOverview: movieData['overview'],
+      voteCount: movieData['vote_count'],
+      voteAverage: movieData['vote_average'].toDouble(),
+      genres: List<int>.from(
+        movieData['genre_ids'],
+      ),
+      originalTitle: movieData['original_title'],
+      dateRelease: movieData['release_date'],
+      movieTitle: movieData['title'],
+      imagePath: 'https://image.tmdb.org/t/p/w500${movieData['poster_path']}',
+      backdropPath:
+          'https://image.tmdb.org/t/p/w500${movieData['backdrop_path']}',
+      language: movieData['original_language'],
+    );
+  }
+
+  static List<MovieModel> listOfMovies(
+    List<dynamic> data,
+  ) {
+    return data
+        .map(
+          (
+            movie,
+          ) =>
+              MovieModel.fromJson(
+            movie,
+          ),
+        )
+        .toList();
+  }
```

Comment:
Why do you need this method?

Reply:
Because in the MoviePageModel, I need the results to be a List<MovieModel>, and the data from the json is a List<dynamic>